app/controllers/conference_registrations_controller.rb

```diff
@@ -33,8 +33,10 @@ def new
   end
 
   def show
-    @total_price = Ticket.total_price(@conference, current_user)
+    TicketPurchase.destroy_all(user_id: current_user.id, conference_id: @conference.id, paid: 'f')
```

Comment:
I am not modifying the existing data in the application with this code, rather , I am handling a case in which, if the user creates a ticket_purchase and somehow fails to complete his payment, (for example, they click the back button or close the webpage), then the `ticket_purchase` record should be deleted as well.